### PR TITLE
feat: video-library screens

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added screens of all the p0 components of the video library with their respective routes
- screens added: homepage, video listing, log in, signup, history, watch later, playlist listing

### CODE REVIEW

File: src/pages/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    return (
+        <main>
+            <section className="login">
+                <form action="" method="get" className="form p-m m-m br-s">
+                    <h1 className="m-s p-s txt-center">Login</h1>
+
+                    <div className="flex-column">
+                        <label htmlFor="Email" className="m-xs" id='email1'>Email Address:</label>
+                        <input type="email" name="Email" className="input-box p-s m-xs br-s" placeholder="abc@xyz.com" id='email1' />
+                    </div>
```
Comment (JantuDeb) [inline src/pages/Login.jsx:15]: You can create a `Input` component

**Review by JantuDeb — APPROVED** [review]
This is looks good to me

File: src/pages/Login.jsx (hunk 1 above)
Comment (harshitadaswani) [reply src/pages/Login.jsx:15]: will refactor that after completing p0 screens

**Review by mariyasada — APPROVED** [review]
The code looks clean. liked that you have defined all the routes with an error page

**Pull request merged**